site/features.md

```diff
@@ -17,17 +17,17 @@ Browse the current feature set for Scope with links to relevant indepth topics:
 
 Scope builds logical topologies of your application and infrastructure.  A topology is a collection of nodes and edges, where nodes can represent objects like processes, container or hosts.  In Scope, edges represent TCP connections between nodes.  Scope displays an intelligent map of your app, so that different node types can be easily viewed and specific microservices can be drilled down on and inspected.
 
-!['Automatic Topology Mapping'](images/topology.png)
+!['Automatic Topology Mapping'](images/topology-map.png)
 
-##<a name="flexible-view-filtering"></a>Flexible Filtering
+##<a name="flexible-filtering"></a>Flexible Filtering
 
 Nodes can be filtered by various properties, for example in the Container view, you can filter by System Containers vs Application Containers or by Stopped vs. Running Containers. Scope can also display various metrics such as CPU and Memory usage in the nodes, for example allowing you to easily find the container using the most CPU or memory.  Mouse-over any of the nodes to view its filtered metric at a glance.
 
 ###<a name="powerful-search"></a> Powerful Search
 
 Powerful search capability enables you to find node types, containers and processes by name, label or even path.  The search functionality supports simple operands so that you can for example, find processes consuming a certain Memory limit or nodes using too much CPU. You can also stack filters to create custom, complex search criterion. 
 
-!['Stack Filters'](images/search-python.png)
```

Comment:
Does this mean that the `search-python.png` image can be removed? If so, please do.

Also, please review if any other images could be removed.